onnxruntime/test/util/default_providers.cc

```diff
@@ -133,8 +133,8 @@ std::unique_ptr<IExecutionProvider> DefaultNupharExecutionProvider(bool allow_un
 // }
 
 std::unique_ptr<IExecutionProvider> DefaultNnapiExecutionProvider() {
-// For any non - Android system, NNAPI will only be used for ort model converter
-// Make it unavailable here, you can still manually append NNAPI EP to session for model conversion
+// The NNAPI EP uses a stub implementation on non-Android platforms so cannot be used to execute a model.
+// Manually append an NNAPI EP instance to the session to unit test the GetCapability and Compile implementation.
 #if defined(USE_NNAPI) && defined(__ANDROID__)
```

Comment:
>defined(USE_NNAPI) && defined(__ANDROID__) [](http://example.com/codeflow?start=4&length=42)

does this condition need to be updated? otherwise it is still only if `defined(__ANDROID__)`